js/navigation/router.js

```diff
@@ -1,9 +1,9 @@
-import { createRouter, } from '@exponent/ex-navigation';
+// import { createRouter, } from '@exponent/ex-navigation';
 
-import Layout from './NavigationLayout';
-import Hello from '../scenes/Hello';
+// import Layout from './NavigationLayout';
+// import Hello from '../scenes/Hello';
 
-export default createRouter(() => ({
-  layout: () => Layout,
-  hello: () => Hello
-}));
\ No newline at end of file
+// export default createRouter(() => ({
+//   layout: () => Layout,
+//   hello: () => Hello
+// }));
```

Comment:
Just make a call on this file. If you don't want it, delete it. Otherwise put some stuff in it to be imported. No big deal, we can always refactor later if needed.